extensions/3DTILES_metadata/README.md

```diff
@@ -0,0 +1,115 @@
+# 3DTILES_metadata Extension
+
+## Version
+
+Version: **Version 0.0.0**
+Revision Date: November 6, 2020.
+
+## Contributors
+
+Cesium
+
+## Status
+
+Draft (version 0.0.0)
+
+## Overview
+
+This extension to 3D Tiles enables the declaration of metadata classes and the assignment of metadata to the tileset. Metadata classes are declared and assigned based on the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/tree/3d-metadata-spec/specification/Metadata). Metadata may also be assigned to layers using [3DTILES_layers](https://github.com/CesiumGS/3d-tiles/tree/3DTILES_layers/extensions/3DTILES_layers). The following diagrams illustrates how these extensions are connected:
```

Comment:
Just an updated reminder: on Friday we'll probably merge `3d-metadata-spec -> 3d-tiles-next`